core-kotlin-modules/core-kotlin-collections-list-2/src/test/kotlin/changeMutableListValue/ChangeMutableListValueUnitTest.kt

```diff
@@ -0,0 +1,41 @@
+package changeMutableListValue
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class ChangeMutableListValueUnitTest {
+
+    @Test
+    fun `changes mutable list value using set method`() {
+        val mutableList = mutableListOf("kotlin", "java", "android")
+        mutableList.set(1, "swift")
+        assertEquals(mutableListOf("kotlin", "swift", "android"), mutableList)
+    }
+
+    @Test
+    fun `changes mutable list value using indexed access operator`() {
+        val mutableList = mutableListOf("kotlin", "java", "android")
+        mutableList[1] = "swift"
+        assertEquals(mutableListOf("kotlin", "swift", "android"), mutableList)
+    }
+
+    @Test
+    fun `changes mutable list value using the map method`() {
```

Comment:
Let's update this test name to make it clear that a new list is created - we aren't really changing the original list here